modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbTableStorage.java

```diff
@@ -153,6 +153,8 @@ public TxStateStorage getOrCreateTxStateStorage(int partitionId) {
                 partitionId,
                 this
             );
+
+            storage.start();
```

Comment:
Technically, we should close the instance if there's an exception in "start()", right? We may fix that in the future. This is the reason why I don't like exceptions in constructors - they may easily produce leaks.

Reply:
Ok.